code/modules/mob/living/carbon/xenomorph/castes/Carrier.dm

```diff
@@ -124,6 +124,15 @@
 	if(.)
 		var/chance = 75
 
+		if (huggers_cur)
+			//Hugger explosion, like an egg morpher
+			var/obj/item/clothing/mask/facehugger/hugger
+			visible_message(SPAN_XENOWARNING("The chittering mass of tiny aliens is trying to escape [src]!"))
```

Comment:
```suggestion
			balloon_alert_to_viewers(SPAN_XENOWARNING("The chittering mass of tiny aliens is trying to escape [src]!"))
```

Reply:
seems like a perfect time to use em

Reply:
I dunno, I don't think we ever use overhead messages like these for narration:
![image](https://user-images.githubusercontent.com/964559/206146010-14b7ea4c-0f19-489a-b399-1abbce2d3c9e.png)

It's an interesting function otherwise!